modules/ztp-additional-worker-to-sno-cluster.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+// Epic CNF-5335 (4.11), Story TELCODOCS-643
+// scalability_and_performance/ztp-deploying-disconnected.adoc
+
+:_content-type: CONCEPT
+[id="ztp-additional-worker-sno"]
+= {sno-caps} cluster expansion with worker nodes
+include::../_attributes/common-attributes.adoc[]
+//:context: additional-worker-sno
+
+You can add one or more worker nodes to an existing SNO cluster to increase CPU resources used by the original SNO control plane node. The addition of worker nodes does not require any downtime for the existing SNO.
```

Comment:
Consider spelling out the first instance of SNO--e.g., Single-node OpenShift (SNO). Consider "the existing SNO cluster."